libctru/source/system/allocateHeaps.c

```diff
@@ -3,24 +3,33 @@
 #include <3ds/env.h>
 #include <3ds/os.h>
 
+#define MAX_LINEAR_HEAP_SIZE (32 << 20) // 32MB
```

Comment:
Rename to `DEFAULT_LINEAR_HEAP_SIZE`, as this name might misleadingly imply that you can't have more than 32mb of linear heap no matter what (and you can override `__ctru_linear_heap_size` with a bigger size).